### ISSUE TITLE
Error connecting to yts.to

### ISSUE THREAD

**Issue by pbassut** [OstlerDev/PopcornTV#238]
So, when accessing the trailers app I saw this on the console:

```warning: Error connecting to yts.to and grabbing json: https://yts.to/api/v2/list_movies.json?sort_by=seeds&limit=50&page=1
Web: /Parade.xml
warning: Error connecting to yts.to and grabbing json: https://yts.to/api/v2/list_movies.json?sort_by=seeds&limit=15&page=1```

And it keeps loading.

**Comment by OstlerDev** [comment]
Can you load that page in your Web Browser?

**Comment by pbassut** [comment]
Yes

**Comment by OstlerDev** [comment]
Can you try closing the application completely and trying again?

**Comment by pbassut** [comment]
```^CPatricks-Mini-2:PopcornTV-master patrickbassut$ sudo node atv.js 
Password:
notice: Starting PopcornTV
DNS: DnsProxy binding on 192.168.25.3:53
Web: listening on 192.168.25.3:80
Web: SSL Web: listening on 192.168.25.3:443
notice: Starting Commit 
Web: templates/MovieNav.xml
Web: /js/utils.js
Web: /js/navbarHandler.js
Web: /MoviesGrid.xml
warning: Error connecting to yts.to and grabbing json: https://yts.to/api/v2/list_movies.json?sort_by=seeds&limit=50&page=1```

**Comment by OstlerDev** [comment]
what does it look like when you load [this](https://yts.to/api/v2/list_movies.json?sort_by=seeds&limit=50&page=1) page?

**Comment by pbassut** [comment]
It looks correct. It's a json with the movie list

**Comment by OstlerDev** [comment]
Hmm, could you try posting all the needed files using [this guide?](https://github.com/OstlerDev/PopcornTV/wiki/How-to-report-an-issue)

I am not sure what it could be.

**Comment by pbassut** [comment]
https://gist.github.com/pbassut/a198a4bd9907e8db7aae - Log
https://gist.github.com/pbassut/f75281b3566b118a577f - settings.js ***(Couldn't find ```aTVSettings.js```)***
https://gist.github.com/pbassut/7501974fc354a5e57af4 - config.json

**Comment by OstlerDev** [comment]
Can you navigate to the settings page before attempting to go into movies? I want to see if that fixes it.

**Comment by pbassut** [comment]
The settings page still in the trailers app, right? And then go to "libraries". 
No, still the same issue.

**Comment by OstlerDev** [comment]
Yeah, thats what I meant. What happens when you go into "TV Shows"

**Comment by pbassut** [comment]
Then the error gets a bit different:
```Web: templates/TVNav.xml
Web: /Parade.xml
Web: /js/utils.js
Web: /js/navbarHandler.js
Web: /TVGrid.xml
warning: Error connecting to eztvapi.re and grabbing json: http://eztvapi.re/shows/1
warning: Error connecting to eztvapi.re and grabbing json: http://eztvapi.re/shows/1```

**Comment by OstlerDev** [comment]
What OS are you using?

It seems like there is an issue with outgoing requests.

**Comment by pbassut** [comment]
Yosemite

**Comment by idanew** [comment]
i think there is a problem also on El capitan
i am getting similar issues
to me it is stuck on "Accessing Trailers" without an error on the terminal window
last week i got error connecting to.... messages lots of them

help needed

**Comment by pbassut** [comment]
@OstlerDev Are you on El Capitan?

**Comment by louisthompsonuk** [comment]
I am also getting the error connecting to yts.to and also cannot access via my browser as it is blocked by my ISP. Should I just scrap using this or is there a way round it?

**Comment by OstlerDev** [comment]
@louisthompsonuk I am going to implement a cloudflare version to get around this hopefully later today.

@pbassut I am on El Capitan and it is working fine for me. I am going to reclone and try again to see if that is the issue.

**Comment by Pamelam76** [comment]
First of all thanks for developing a way to access to popcorn from an apple tv :D
I have a little issue,my apple tv doesnt allow me to add the trailers profile
 I've checked twice if I have the correct DNS address and that's not the problem
 what I'm doing wrong?

**Comment by pbassut** [comment]
@Pamelam76 This issue is totally unrelated to your problem. You should create a new issue for that.
And there you also specify the error a little better, cause "Doesn't allow to add the trailers profile" won't be very helpful.

If you're using a different remote though, that might be your issue. Try apple's remote.

**Comment by felipemarcondes** [comment]
Hi guys, Im having the same exact problem.

**Comment by OstlerDev** [comment]
@felipemarcondes @pbassut Make sure that you are running Node v0.12.x. The new versions of 4.x.x are not supported yet due to them changing a lot of the code.

**Comment by pbassut** [comment]
I am...
![image](https://cloud.githubusercontent.com/assets/1500037/10473084/86e4271c-71fd-11e5-970d-b3918b30d171.png)

**Comment by felipemarcondes** [comment]
It worked for me. I installed the v0.12.0

**Comment by pbassut** [comment]
Installed v0.12.0 and still getting the very same error. ;(

**Comment by OstlerDev** [comment]
@pbassut Are you from the UK? If so, try changing your TVEndpoint in the settings.

**Comment by pbassut** [comment]
@OstlerDev I'm from Brazil, actually... Do I have to change anything on the settings at all? well, that might be it.

**Comment by OstlerDev** [comment]
That might be it, what happens when you go to http://eztvapi.re/shows/1 in your browser? If it times out then you need to change the API (in the settings).

**Comment by pbassut** [comment]
It does not time out, unfortunately.

**Comment by OstlerDev** [comment]
@pbassut check your firewall in Mac. There could be an issue where you have a firewall turned on that is blocking PopcornTV.

**Comment by pbassut** [comment]
Firewall is off.

**Comment by pbassut** [comment]
Also, I think it would timeout when I used the browser. Unless I applied some configs specifically to your application.

**Comment by OstlerDev** [comment]
Are you able to use Popcorn Time (from popcorntime.io) just fine?

**Comment by pbassut** [comment]
![image](https://cloud.githubusercontent.com/assets/1500037/10474479/907c11bc-720d-11e5-916f-f5755bd413bd.png)
Yup. =/
